dex/src/fees.rs

```diff
@@ -94,7 +94,7 @@ impl FeeTier {
         match self {
             Stable => fee_tenth_of_bps(5),
             Flagship => fee_tenth_of_bps(20),
-            Base | _ => fee_tenth_of_bps(100),
+            Base | _ => fee_tenth_of_bps(40),
```

Comment:
@mschneider 3bps is 30 instead of 40, no?

Reply:
oh yeah, i though this would be a reasonable compromise, but also happy to go with this.